app/models/similarity_metric.rb

```diff
@@ -146,8 +149,10 @@ def single_score(c1, c2, metrics: Base.usable.all.reject(&:bogus?), just_score:
       end
     end.compact
     s = if scores.any?
+      # the weighted average of all the scores
       scores.sum / weights.map(&:weight).sum
```

Comment:
Is `.sum(&:weight)` different from `.map(&:weight).sum`? (lines 139 and 152 handle that differently)

Reply:
It's an accidental difference in implementation with no effect. I'll fix it so they're the same.